dpnp/tests/helper.py

```diff
@@ -37,6 +37,12 @@ def assert_dtype_allclose(
 
     """
 
+    if isinstance(numpy_arr, numpy.ndarray):
```

Comment:
We are overriding `assert_allclose` in `dpnp/tests/testing/array.py`. Should we add shape comparison there in order to cover all places where we use `assert_allclose`, not only for `assert_dtype_allclose`?
And probably do the same for `assert_almost_equal` and `assert_array_almost_equal`.
While for `assert_array_equal` and `assert_equal` the shape check is already implemented by numpy f/w, I believe.